assets/js/blocks/classic-template/editor.scss

```diff
@@ -51,7 +52,15 @@
 	align-items: center;
 	gap: $gap-small;
 
+	@media only screen and (max-width: 480px) {
+		gap: $gap-smaller;
+	}
```

Comment:
Are you sure that it is necessary? I think that we can remove it.

Reply:
@gigitux same as above, the amount of gap was a lot between the icon and the title (after reducing font size) for mobile. Do you think it is unnecessary?